### PR TITLE
Update wart remover

### PR DESCRIPTION
**Pull request by sellout**

### CODE REVIEW

**Comment by sellout** [comment]
There’s a new comment on puffnfresh/wartremover#226 that may allow me to enable the ExplicitImplicitTypes wart, but I’m not sure if the cure is worse than the disease in this case. Thoughts?

File: core/src/main/scala/quasar/sql/parser.scala [hunk 1]
```diff
@@ -41,6 +41,11 @@ object ParsingError {
 
 final case class Query(value: String)
 
+sealed trait DerefType[T[_[_]]]
```
Comment (drostron) [inline core/src/main/scala/quasar/sql/parser.scala]: Did this sneak in or was it just something you noticed needing updating while passing though?

**Comment by drostron** [comment]
Great to see WartRemover 1.0.0 in the mix. Other than the one question, 👍🏽.

**Comment by drostron** [comment]
Oh, missed your earlier comment — I'd say let's leave the wart disabled and keep an eye on the issue.

File: core/src/main/scala/quasar/sql/parser.scala (hunk 1 above)
Comment (sellout) [inline core/src/main/scala/quasar/sql/parser.scala]: Needed to extract it from the class – WartRemover doesn't like path-dependent types now.

Comment (drostron) [inline core/src/main/scala/quasar/sql/parser.scala]: Ah, interesting. OK.

**Pull request closed without merging**